<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Involute Gear Simulator</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>162</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Use mouse to&lt;br/&gt;reorientate&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Pressure Angle&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>162</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Press pause to&lt;br/&gt;change parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Number of teeth for&lt;br/&gt;each of the two gears &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="speedScrollBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>7</number>
   </property>
   <property name="sliderPosition">
    <number>7</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Light Position</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>750</y>
     <width>87</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pausePlayButton">
      <property name="text">
       <string>&amp;Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>&amp;Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fullScreenButton">
      <property name="text">
       <string>&amp;Full Screen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aboutButton">
      <property name="text">
       <string>&amp;About</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>&amp;Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>81</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="text">
       <string>14.5 deg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_20">
      <property name="text">
       <string>20 deg</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_25">
      <property name="text">
       <string>25 deg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>54</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSpinBox" name="spinBox_Na">
      <property name="minimum">
       <number>8</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_Nb">
      <property name="minimum">
       <number>8</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>x: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="lightPosX">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>-500</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>y: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="lightPosY">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>-500</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>z: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="lightPosZ">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Seperation error</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SeperationSpinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>0.750000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="instructionsButton">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>60</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Instructions</string>
   </property>
  </widget>
  <widget class="QLabel" name="toggleLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>105</y>
     <width>162</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10.5pt; font-weight:600;&quot;&gt;Exact Involute&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleButton">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>130</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Toggle</string>
   </property>
  </widget>
  <widget class="OGLWidget" name="myOGLWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>1200</width>
     <height>900</height>
    </rect>
   </property>
  </widget>
  <zorder>layoutWidget4</zorder>
  <zorder>layoutWidget3</zorder>
  <zorder>layoutWidget2</zorder>
  <zorder>layoutWidget1</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>speedScrollBar</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>SeperationSpinBox</zorder>
  <zorder>instructionsButton</zorder>
  <zorder>toggleLabel</zorder>
  <zorder>toggleButton</zorder>
  <zorder>myOGLWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>oglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
